src/css/partials/_styles.scss

```diff
@@ -111,82 +115,86 @@ img {
   background-color: $active-color;
 }
 
-.all {
+.all { // not sure what this is styling; cant find a class named "all"; does this apply to everything that contains the word "all" in the classList?
```

Comment:
This styles the recipe detail view - all gets added to the card area class. We can chat about it!